### ISSUE TITLE
[Bug] Test fails due to permission issue in CI

### ISSUE THREAD

**Issue by kyungeunni** [elastic/synthetics-recorder#338]
### Bug summary

When running test in CI, it fails as the file read or write can't be perfromed due to permission issues.
Most recent example can be found [here](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-rum%2Fsynthetics-recorder-mbp/detail/PR-337/1/pipeline#step-277-log-214) and here's the screenshot
<img width="810" alt="image" src="https://user-images.githubusercontent.com/16660866/209285324-7afd1a47-ab66-4009-8309-f01bf71717be.png">

cc @elastic/observablt-robots @v1v 

### Recorder Version

N/A

### Steps to reproduce

Run tests

**Comment by kuisathaverat** [comment]
@reakaleek check if the test run in a Docker container that uses the root user, if so, change it to use the local user with `-u $(id -u):$(id -g)`

**Comment by reakaleek** [comment]
Thank you for the tip. I had the same idea!

Pretty sure this is the issue, the tests ur running in a docker container, just as you presumed.

However, I'm currently struggling with another issue.

While trying to reproduce it, my [PR](https://github.com/elastic/synthetics-recorder/pull/339) already fails at the install step.

```
pm ERR! Cannot read properties of null (reading 'pickAlgorithm')
```

I was able to get the same error locally.

I tried multiple things like

`npm cache clear --force` or `npm install` instead of `npm ci`.

The only approach that worked is `npm update` but this updates all the packages in the `package-lock.json` file.

I will do it for now so I can continue..

**Comment by reakaleek** [comment]
@kuisathaverat Thank you for the tip. I had the same idea!

Pretty sure this is the issue, the tests ur running in a docker container, just as you presumed.

However, I'm currently struggling with another issue.

While trying to reproduce it, my [PR](https://github.com/elastic/synthetics-recorder/pull/339) already fails at the install step.

```
pm ERR! Cannot read properties of null (reading 'pickAlgorithm')
```

I was able to get the same error locally.

I tried multiple things like

`npm cache clear --force` or `npm install` instead of `npm ci`.

The only approach that worked is `npm update` but this updates all the packages in the `package-lock.json` file.

I will do it for now so I can continue..
